src/Tizen.Multimedia/Common.Internal/ObjectKeeper.cs

```diff
@@ -63,6 +64,7 @@ public override void Dispose()
             {
                 Dispose(true);
                 GC.SuppressFinalize(this);
+                Tizen.Log.Warn("Tizen.Multimedia", "GCHandle is disposed.");
```

Comment:
It's not warning situation. After lifecycle of ObjectKeeper is done, ```using``` statement will call Dispose() or user have to call it explicitly to free GCHandle.
Please change Warn to Info.